perfdash/parser.go

```diff
@@ -137,3 +137,58 @@ func parseRequestCountData(data []byte, buildNumber int, testResult *BuildData)
 		}
 	}
 }
+
+type metricValue struct {
+	Metric map[string]string `json: metric`
+	Value  []interface{}     `json: value`
+}
+
+type e2eMeteric []metricValue
+
+type e2eMetrics struct {
+	ApiServerMetrics         map[string]e2eMeteric `ApiServerMetrics`
+	ClusterAutoscalerMetrics map[string]e2eMeteric `ClusterAutoscalerMetrics`
+}
+
+func createApiserverRequestCountData(data *perftype.DataItem, metric *metricValue) error {
+	data.Unit = ""
+	data.Data = make(map[string]float64)
+	data.Labels = metric.Metric
+	delete(data.Labels, "__name__")
```

Comment:
I think you probably can also get rid of `contentType` label - it's not too useful IMO.

Reply:
done